ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly-4.15-upgrade-from-stable-4.5.yaml

```diff
@@ -98,13 +98,20 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: gcp-ipi-proxy-f30
+- as: aws-ipi-f30
   cron: 5 4 24 * *
   steps:
-    cluster_profile: gcp-qe
+    cluster_profile: aws-qe
```

Comment:
Is this update to match the [chain upgrade matrix](https://docs.google.com/spreadsheets/d/12NAZtZas0cHBjO3BlpdM_AuUaoYin7Na68K7SrEHvkA/edit#gid=0) for 4.5-4.15 upgrade?

Reply:
Not really. The 4.5 fresh install on GCP will fail due to https://issues.redhat.com/browse/OCPQE-7898. So switching it to aws.

Reply:
ack, i will add it to the sheet as a change log